coverage/.resultset.json

```diff
@@ -0,0 +1,420 @@
+{
```

Comment:
add /coverage/ folder to .gitignore and exclude it from this pr,
Those files will be always changed and it is very hard to resolve conflicts it coverage files